src/Agent.Plugins/BuildArtifact/BuildArtifactPluginV1.cs

```diff
@@ -307,6 +318,39 @@ private string CreateDirectoryIfDoesntExist(string targetPath)
             return fullPath;
         }
 
+        private void CleanDirectory(AgentTaskPluginExecutionContext context, string directoryPath)
+        {
+            context.Output(StringUtil.Loc("CleaningDestinationFolder", directoryPath));
+
+            FileAttributes dirAttributes = File.GetAttributes(directoryPath);
```

Comment:
Will File.GetAttributes throw not found exception according to [docs](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.getattributes?view=net-5.0) if path does not exist? If so, it would be inconsistent with node handler behavior.

Reply:
Actually, I call this function after "CreateDirectoryIfDoesntExist" so this folder should exist anyway, but I'll add exception handling.